tutorials/tutorial3/tutorial_3.rb

```diff
@@ -0,0 +1,51 @@
+puts "Enter a number to calculate: "
+num1 = gets.chomp().to_f
+
+puts "Enter operator to calculate (Eg. + , - ,*, /,% ): "
+op = gets.chomp()
+
+puts "Enter another number to calculate: "
+num2 = gets.chomp().to_f
+
+if op == "+"
+  puts (num1 + num2)
+elsif op == "-"
+  puts (num1 - num2)
+elsif op == "*"
+  puts (num1 * num2)
+elsif op == "/"
+  puts (num1 / num2)
+elsif op == "%"
+  puts (num1 % num2)
+else
+  puts "Invalid operator"
+end
+puts "Would you like to calculate another operation?(Please enter yes if you want to continue : yes (or) no ) "
+con = gets.chomp()
+
```

Comment:
I think line no 1 to 24 do not need. Instead of using these code, you should use only this code line.
con = "yes"

Reply:
Thanks ,Bro @HtetTunLu .I committed the code. Here is the commit : [b012e87](https://github.com/maythuzarlinn/RUBY_TRAINING/pull/5/commits/b012e8771b17273b76f71de196b9eb4efe339d5e)